.github/actions/veracode/source_clear.sh

```diff
@@ -3,7 +3,6 @@
 echo "=========================== Starting SourceClear Script ==========================="
 PS4="\[\e[35m\]+ \[\e[m\]"
 set +e -v -x
-pushd "$(dirname "${BASH_SOURCE[0]}")/../"
```

Comment:
Since we removed the `pushd` instruction here, shouldn't we also remove the `popd` at L#21, or what purpose does it serve at this point?

Reply:
Good point, we don't need the `popd` instruction, I removed it.